examples/Playground.hs

```diff
@@ -43,8 +43,7 @@ showSomeLog = do
 main :: IO ()
 main = do
     testToJsonConfigOutput
-    let config = (productionB <> prefixB "logs")
     bracket_
-        (buildAndSetupYamlLogging config testLoggerConfigPath)
+        (buildAndSetupYamlLogging productionB testLoggerConfigPath)
         removeAllHandlers
```

Comment:
I forgot about one thing: users of `log-warper` **MUST** call `removeAllHandlers` function at the end of logging action. Could you please move `bracket_` to `launch*` functions?